### PR TITLE
Fix missing part sfor the automated upgrades

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
The automated rust and didc upgrades don't work yet.

# Changes
* Fix the code that gets the current didc version.
* Add the slack integration (still needs a secret).

# Tests
This cannot be tested properly until we have a slack token.

# Todos

- [x] Add entry to changelog (if necessary).
  There is  already a changelog entry for this integration.

### CODE REVIEW

File: .github/actions/slack/action.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: 'Slack'
+description: 'A simple action that send a slack message to a channel.'
```
Comment (mstrasinskis) [inline .github/actions/slack/action.yml:2]:
```suggestion
description: 'A simple action that sends a slack message to a channel.'
```

**Review by mstrasinskis — APPROVED** [review]
Rubber stamp, thanks!

**Review by peterpeterparker — APPROVED** [review]
Rubber stamp

File: .github/actions/slack/action.yml (hunk 1 above)
Comment (bitdivine) [reply .github/actions/slack/action.yml]: Even a real review!  Thanks.  Done.

**Pull request merged**